.github/workflows/src/summarize-checks/summarize-checks.js

```diff
@@ -319,7 +319,17 @@ export async function summarizeChecksImpl(
   event_name,
   targetBranch,
 ) {
-  core.info(`Handling ${event_name} event for PR #${issue_number} in ${owner}/${repo}.`);
+  core.info(
+    `Handling ${event_name} event for PR #${issue_number} in ${owner}/${repo}@${head_sha}.`,
+  );
+
+  // == null covers === undefined implicitly
+  if (issue_number == null || isNaN(issue_number) || issue_number <= 0) {
```

Comment:
```suggestion
  if (issue_number) {
```

Checking for truthiness is simpler and should be sufficient.  I also think this should move into summarizeChecks(), since it's closely related to extractInputs().